elixir/apps/api/test/api/gateway/channel_test.exs

```diff
@@ -317,4 +325,55 @@ defmodule API.Gateway.ChannelTest do
       assert gateway.id == gateway_id
     end
   end
+
+  describe "handle_in/3 metrics" do
+    test "inserts activities", %{
+      account: account,
+      subject: subject,
+      client: client,
+      gateway: gateway,
+      resource: resource,
+      socket: socket
+    } do
+      flow =
+        Fixtures.Flows.create_flow(
+          account: account,
+          subject: subject,
+          client: client,
+          resource: resource,
+          gateway: gateway
+        )
+
+      now = DateTime.utc_now() |> DateTime.truncate(:second)
+      one_minute_ago = DateTime.add(now, -1, :minute)
+
+      {:ok, destination} = Domain.Types.IPPort.cast("127.0.0.1:80")
+
+      attrs =
```

Comment:
@conectado this is a sample payload so you can see the structure. Destination is a binary, `address:port` for IPV4, `[address]:port` for IPV6.